.github/workflows/ci.yml

```diff
@@ -19,9 +19,7 @@ jobs:
       matrix:
         # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
         os: [ubuntu-20.04, windows-2019, macos-10.15]
-        # node-version: [16.x, 14.x, 12.x]
-        # FIXME: find out why Node 16 crashes.
```

Comment:
This is still crashing: https://github.com/inrupt/solid-client-js/runs/4537142263?check_suite_focus=true

Reply:
It's an env variable problem, I'll fix it.